whisper/transcribe.py

```diff
@@ -170,7 +172,7 @@ def add_segment(
     num_frames = mel.shape[-1]
     previous_seek_value = seek
 
-    with tqdm.tqdm(total=num_frames, unit='frames', disable=verbose) as pbar:
+    with tqdm.tqdm(total=num_frames, unit='frames', disable=verbose is False) as pbar:
```

Comment:
Could you make this `verbose is not False`? Thank you.

Reply:
Done